src/frontend/web_application/src/scenes/Timeline/components/MessageItem/presenter.jsx

```diff
@@ -151,20 +162,13 @@ class MessageItem extends Component {
             />
           </label>
         </div>
-        <div className="s-message-item__col-title">
-          {this.renderTitle()}
-        </div>
-        <div className="s-message-item__col-file">
-          { attachments && attachments.length !== 0 && <Icon type="paperclip" /> }
+        <div className="s-message-item__column--content">
+          {this.renderContent()}
         </div>
-        <div className={classnames(
-          's-message-item__col-dates',
-          { 's-message-item__col-dates--unread': message.is_unread }
-          )}
-        >
+        <div className="s-message-item__column--date">
           {this.renderDate()}
         </div>
-        <div className="s-message-item__col-select">
```

Comment:
the tests fails because this className was used as a selector:

```
Failed: No element found using locator: By(css selector, .s-message-item__col-select input[type=checkbox])
```